controller/users.go

```diff
@@ -130,14 +130,14 @@ func (c *UsersController) Obfuscate(ctx *app.ObfuscateUsersContext) error {
 				"user_id": u,
 				"err":     err,
 			}, "unable to retrieve the identity associated to this user id")
-			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err))
+			return err
```

Comment:
Don't you wanna return a NotFound error here anymore?

Reply:
Here I want to load by the foreign key therefore i use the `Query`, but I'll change to return the NotFoundError

Reply:
fixed in b5abde4